app/controllers/mixins/ems_common_angular.rb

```diff
@@ -134,7 +134,7 @@ def ems_form_fields
       keystone_v3_domain_id = ""
       hawkular_hostname = ""
       hawkular_api_port = ""
-      hawkular_security_protocol = ""
+      hawkular_security_protocol = security_protocol_default
```

Comment:
@cben Even though you are setting the default value for `hawkular_security_protocol` here, why is it that the form does not show it?

<img width="1142" alt="screen shot 2017-03-15 at 12 24 52 pm" src="https://cloud.githubusercontent.com/assets/1538216/23967106/eb2db434-097a-11e7-9f95-260f1e25679d.png">